hopsworks-common/src/main/java/io/hops/hopsworks/common/user/UsersController.java

```diff
@@ -386,46 +387,39 @@ public void removeSshKey(int id, String name) {
   }
 
   public String generateUsername(String email) {
-    Integer count = 0;
-    String uname = getUsernameFromEmail(email);
-    Users user = userFacade.findByUsername(uname);
-    String suffix = "";
+    String emailUsername = email.substring(0, email.lastIndexOf("@")).toLowerCase();
```

Comment:
@SirOibaf are we sure this will never be invoked with a null email?

Reply:
The email is validated by the `UserValidator` class. But I can add a check just in case the method is called by somebody else.